### ISSUE TITLE
Vlastnictví, Locke a "zbyde dost pro ostatní"

### ISSUE THREAD

**Issue by hocker1** [galtalex666/anarchocapitaliscam#1]
Mám ve Vašem textu podezření na logický kiks. Konkrétně když z:

> Nikdo jiný než on pak nemá právo na to k čemu svou práci připojil, přinejmenším tam, kde je ponecháno dost a stejně dobrého, co zbývá pro ostatní.

... vyvozujete:

> Vlastnictví je tedy zásadně podmíněno tím, že zbývá dost pro ostatní...

Z Lockovy formulace plyne, že _když zbyde dost pro ostatní, pak nikdo jiný nemá právo na můj majetek_. Neplyne z toho ale nutně opak, tedy že _když nezbyde dost pro ostatní, tak mají právo na moje_. Neboli z tvrzení `B => A` nevyplývá `!B => !A` (v tomto případě `A` je "nikdo nemá právo na statky, které jsem smísil se svou prací", `B` je "zbylo dost pro všechny").

Prosím o bugfix, bude-li issue uznáno bugem :-)

**Comment by galtalex666** [comment]
Ano, Locke říká, že jediný případ, kdy je to jasné je: *když zbyde dost pro ostatní, pak nikdo jiný nemá právo na můj majetek*. Případ, kdy nezbývá dost pro ostatní tedy není jasný a může buď také platit, že nikdo jiný nemá právo na můj majetek nebo i to, že k němu nemám vůbec žádná práva.

Já zastávám váklad, že vlastnictví trvá i když nezbývá dost pro ostatní, ale jeho práva jsou omezena.

Text upravím způsobem, aby bylo jasné, že to co následuje je můj výklad.
